tools/sql_extraction/src/main/kotlin/Cli.kt

```diff
@@ -74,13 +72,28 @@ private class Cli(
         "--pretty", help = "Pretty-print output JSON"
     ).flag()
 
+    private val showProgress: Boolean by option(
+        "--progress", help = "Print progress to STDERR"
+    ).flag()
+
+    private val debug: Boolean by option(
+        "--debug", "--verbose", help = "Print debug logs to STDERR"
```

Comment:
Do we have to specify both options to make it work?

Reply:
It works for either one